src/server/services/procedures/roboscape/robot.js

```diff
@@ -622,6 +642,25 @@ Robot.prototype.playBlinks = function (states) {
     setTimeout(step, 0);
 };
 
+/**
+ * Send a list of numbers to a robot
+ * @param {Array<Number>} states 
+ * @param {Number} delay Time each number should be shown for 
+ */
+Robot.prototype.playNumbers = function (states, delay = 4000) {
+    var myself = this,
+        index = 0,
+        step = function () {            
+            if (index < states.length) {
+                myself.setNumericDisplay(states[index]);
+                index += 1;
+                setTimeout(step, delay);
```

Comment:
Should this clear the display at the end? Or will the last number be showing perpetually? Is there a way for the user to clear it?